tests/installation/installer_desktopselection.pm

```diff
@@ -36,6 +42,12 @@ sub run() {
     send_key 'spc';                                                            # Select the desktop
 
     assert_screen "$d-selected";
+    if (check_var('VERSION', 'Tumbleweed')) {
```

Comment:
I don't think there's a need to pretend we know it works against pie in the sky version.